src/test/java/org/broadinstitute/consent/http/db/UserDAOTest.java

```diff
@@ -169,22 +169,6 @@ public void testInsertDACUser() {
         // No-op ... tested in `createUser()`
     }
 
-    @Test
-    public void testUpdateDACUser_case1() {
-        User user = createUser();
-        Institution firstInstitute = createInstitution();
-        String newEmail = getRandomEmailAddress();
-        userDAO.updateUser(
-                "Dac User Test",
-                user.getUserId(),
-                newEmail,
-                firstInstitute.getId()
-                );
-        User user2 = userDAO.findUserById(user.getUserId());
-        assertEquals(user2.getAdditionalEmail(), newEmail);
-        assertEquals(user2.getInstitution().getId(), firstInstitute.getId());
-    }
```

Comment:
I think we still need this test case,  but without the email component. This test is actually testing two cases, institution and email updates. We should maintain the institution part of this test. I do see usages in the helper class, but that's not sufficient - we should have a test for `userDAO.updateUser` here too.

Reply:
ahh, didn't see the institution change - brought the test back!